block/manager.go

```diff
@@ -295,6 +295,21 @@ func (m *Manager) GetStoreHeight() uint64 {
 	return m.store.Height()
 }
 
+// GetDAIncludedHeight returns the manager's da included height
+// FIXME(tuxcanfly): this is in accurate before first retrieve loop
```

Comment:
@tzdybal we store hard confirmations only in block cache, not store. Do you think we should persist it there as well so we don't have to wait for the retrieve loop to be able to the return the current hard confirmed head?

Reply:
Reflecting on this, we can have an option to force a rescan of the DA chain to get the latest confirmed heads. Even with a persistent store, there will be a "stale" period before the rescan where the current head will be outdated.

Reply:
More thoughts after discussing with @Manav-Aggarwal : this can be a DoS vector. An alternative is to return last scanned DA height in the response, so the consumer can know that the data is potentially stale and retry after a while.

Reply:
Made it persistent to store as metadata. There are some stale periods but I think it's fine as the consumer can simply retry.